src/libraries/Microsoft.Extensions.Configuration.Binder/gen/KnownTypeSymbols.cs

```diff
@@ -26,19 +22,29 @@ internal sealed record KnownTypeSymbols
         public INamedTypeSymbol? Uri { get; }
         public INamedTypeSymbol? Version { get; }
 
-        public INamedTypeSymbol? Action { get; }
         public INamedTypeSymbol? ActionOfBinderOptions { get; }
-        public INamedTypeSymbol? BinderOptions { get; }
         public INamedTypeSymbol? ConfigurationKeyNameAttribute { get; }
-        public INamedTypeSymbol? Dictionary { get; }
+
+        public INamedTypeSymbol GenericIList { get; }
+        public INamedTypeSymbol GenericICollection { get; }
+        public INamedTypeSymbol GenericICollection_Unconstructed { get; }
+        public INamedTypeSymbol GenericIEnumerable { get; }
+        public INamedTypeSymbol IEnumerable { get; }
+        public INamedTypeSymbol? Dictionary_Unconstructed { get; }
```

Comment:
What does "unconstruced" mean in this context? is it the generic type definition?

Reply:
Close. I'll rename & add a comment.

Reply:
I hope https://github.com/dotnet/runtime/pull/86285/commits/c4a363539de26f1cd202244cbc044166594bac76 helps clarify.